Tutorials/intro-to-timing/example1.py

```diff
@@ -0,0 +1,35 @@
+
+from qm.QuantumMachinesManager import QuantumMachinesManager
+from qm.qua import *
+from qm import SimulationConfig
+import matplotlib.pyplot as plt
+from configuration import *
+
+QMm = QuantumMachinesManager()
+
+with program() as prog:
+    with program() as prog:
```

Comment:
with program inside with program
Does this works? if so, we need to open an issue in the sdk github

Reply:
Oh ... this is by accident but yes it works ... 
Should I open an issue?

Reply:
Yes, here:
https://quantum-machines.atlassian.net/jira/software/c/projects/QMQUA/boards/72